app/controllers/api/v1/tasks_controller.rb

```diff
@@ -19,7 +19,8 @@ def show
             end
 
             def create
```

Comment:
I think also rename this so more specific e.g. `create_task_and_user` as later we will need a `create` just for making one task (when they're already logged in)